### ISSUE TITLE
WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (.service-page)

### ISSUE THREAD

**Issue by geraldo128** [geraldo128/testMe#2]
#### Issue
`Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds` ([color-contrast](https://dequeuniversity.com/rules/axe/4.0/color-contrast?application=msftAI))
#### Target application
[Django Web Development Company | Durham, NC | Caktus Group](https://www.caktusgroup.com/)
#### Element path
.service-page
#### Snippet
`<a class="service-page" href="/services/">`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 2.68 (foreground color: #94a19f, background color: #ffffff, font size: 13.1pt (17.424px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 86.0.4240.193
This accessibility issue was found using Accessibility Insights for Web 2.22.0 (axe-core 4.0.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
